src/utils/compareDates.js

```diff
@@ -1,17 +1,21 @@
+const MS_TO_SEC = 1000;
+const SEC_TO_MIN = 60;
+const MIN_TO_HR = 60;
+const HR_TO_DAY = 24;
+
+const MS_TO_DAY = MS_TO_SEC * SEC_TO_MIN * MIN_TO_HR * HR_TO_DAY;
+
 /*
 This function receives a Date JS Object and returns 
-true IF it has not passed more than one day, otherwise returns
-false IF it has passed more than 1 day
+an absolute rounded number indicating the difference
+between those two dates in days.
 */
 export const compareDates = (date) => {
```

Comment:
I believe you might rename this function `compareDates` because **_after your refactor AND change_** the logic this function you aren't comparing. You are getting or calculating the difference between two dates.

Maybe a best name could be `diffDateInDays` or `diffDaysBetweenToday` or similar names. 😄 .

Reply:
On it! This would make the function more explicit and we like id :)